apps/docs/app/ui/components/overview/Overview.tsx

```diff
@@ -61,36 +65,40 @@ const Overview = () => {
                     interactive
                     className="hd-component-overview-category__title"
                 >{category}</Title>
-                <div className="hd-component-overview">
-                    {allComponents.filter(component =>
-                        component.category &&
+                <HopperProvider colorScheme={theme} >
```

Comment:
More of a question for curiosity. Why add the HopperProvider here?

Reply:
Hopper tokens are not widely available in the documentation, only where needed, as the illustrations are made using Hopper tokens, I've added the provider in order to be able to use them.